WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/GetDiscoverCardsUseCase.kt

```diff
@@ -24,51 +25,54 @@ class GetDiscoverCardsUseCase @Inject constructor(
     private val parseDiscoverCardsJsonUseCase: ParseDiscoverCardsJsonUseCase,
     private val readerDiscoverCardsTableWrapper: ReaderDiscoverCardsTableWrapper,
     private val readerPostTableWrapper: ReaderPostTableWrapper,
+    private val readerBlogTableWrapper: ReaderBlogTableWrapper,
     private val appLogWrapper: AppLogWrapper,
     private val appPrefsWrapper: AppPrefsWrapper,
     @Named(IO_THREAD) private val ioDispatcher: CoroutineDispatcher
 ) {
-    suspend fun get(): ReaderDiscoverCards =
-            withContext(ioDispatcher) {
-                val cardJsonList = readerDiscoverCardsTableWrapper.loadDiscoverCardsJsons()
-                val cards: ArrayList<ReaderDiscoverCard> = arrayListOf()
+    suspend fun get(): ReaderDiscoverCards = withContext(ioDispatcher) {
```

Comment:
I moved `withContext(ioDispatcher)` up, as the indentation in this file was starting to make it hard to fit anything in one line.